plugin-fixes.php

```diff
@@ -173,3 +173,35 @@ function vip_proxy_twitter_embed( $provider ) {
 	return str_replace( 'https://publish.twitter.com/oembed', 'https://x-oembed-proxy.wpvip.com/oembed', $provider );
 }
 add_filter( 'oembed_fetch_url', 'vip_proxy_twitter_embed', 10 );
+
+/**
+* Ninja Forms (https://wordpress.org/plugins/ninja-forms/)
+*
+* Workaround for a type juggling issue in the Ninja Forms plugin update logic. Valid as of Ninja Forms v3.4.34.2.
+*
+* The plugin uses a loose type comparison when checking option values during updates.
+* Specifically, `update_option()` performs a strict `!==` check, and due to Ninja Forms
+* setting an integer instead of a string (or vice versa), it causes unnecessary DB writes
+* on every wp-admin request.
+*
+* This can lead to excessive `options` table churn and potential `alloptions` bloat,
+* especially on high-traffic editorial sites, triggering performance issues related to roles and rewrites.
+*
+* @see: https://plugins.trac.wordpress.org/browser/ninja-forms/trunk/ninja-forms.php#L541
+* @param mixed $value The new value to be saved.
+* @param string $old_value The old value to be replaced.
+* @return mixed The value to be saved.
+*/
+function vip_ninja_forms_update_option( $value, $old_value ) {
+	return ( 0 === $value && '0' === $old_value ) ? '0' : $value;
+}
+
+function vip_ninja_forms_setup() {
+	if (
```

Comment:
Nitpick, but since this function would need to be called all the time, I think we can just add the checks directly inside the filter since the filter will only run when updating that specific Ninja Forms option -- this will avoids needing to conditionally hook it at load time.